Game/Config/DefaultEditorPerProjectUserSettings.ini

```diff
@@ -1,6 +1,12 @@
 [ContentBrowser]
 ContentBrowserTab1.SelectedPaths=/Game/ThirdPersonCPP
 
+[/Script/UnrealEd.LevelEditorPlaySettings]
+PlayNetMode=PIE_Client
+RunUnderOneProcess=True
+PlayNetDedicated=True
+PlayNumberOfClients=2
```

Comment:
Will this still be required with Nuno's change to the engine?

Reply:
Not the number of clients no - his work is still under review though (https://github.com/improbableio/UnrealEngine/pull/550)

Reply:
Nuno just merged; however on second thought, this is actually still required, as the tests we have are reliant on >1 client. So, unfortunately, it has to stay.